files/fr/web/api/css_counter_styles/index.html

```diff
@@ -0,0 +1,31 @@
+---
+title: Module CSS Styles de compteurs
```

Comment:
I'm torn between the English "canonical term" for the name of the spec and the French equivalent here.
Maybe adding '(Counter Styles)' (within parenthesis) afterwards is a good compromise. I wonder how (French-speaking) people will discover this page: will they search for "counter style css" in their search engine, etc.

Let me know if you disagree or think of a better solution.

Reply:
Agreed. Let's add both, then. New proposal in https://github.com/mdn/translated-content/pull/1766/commits/b819ee39bcf12645dec134db59438b571dbf2f48